kubernetes/istio_details_service.go

```diff
@@ -192,6 +194,30 @@ func (in *IstioClient) getRbacResources() map[string]bool {
 	return *in.rbacResources
 }
 
+func (in *IstioClient) hasSecurityResource(resource string) bool {
+	return in.getSecurityResources()[resource]
+}
+
+func (in *IstioClient) getSecurityResources() map[string]bool {
+	if in.securityResources != nil {
+		return *in.securityResources
+	}
+
+	securityResources := map[string]bool{}
+	resourceListRaw, err := in.k8s.RESTClient().Get().AbsPath("/apis/security.istio.io/v1beta1").Do().Raw()
```

Comment:
minor I wonder if we can reuse constant
```
ApiSecurityVersion
```
where you define the 
```
security.istio.io/v1beta1
```

Just to avoid propagate versions of API when we declared that constant to simplify.

It's minor, just a comment.